web_generator/bin/update_idl_bindings.dart

```diff
@@ -161,18 +163,13 @@ final _endComment =
     '<!-- END updated by $_scriptPOSIXPath. Do not modify by hand -->';
 
 final _usage = '''
-Global Options:
-${_parser.usage}
+${ansi.styleBold.wrap('WebIDL Gen')}:
+$_thisScript [... .idl file] [options]
```

Comment:
We can get rid of this if use a multi-option and make it explicit that if that option is not used, we generate the definitions for `package:web`.